ws-client/src/client.rs

```diff
@@ -102,14 +102,34 @@ pub struct WsClient {
 	id_guard: RequestIdGuard,
 }
 
-/// Configuration.
+/// Builder for [`WsClient`].
+///
+/// # Examples
+///
+/// no_run```
+///
+/// use jsonrpsee_ws_client::{WsClientBuilder, Header};
+///
+/// #[tokio::main]
+/// async fn main() {
+///     // build client
+///     let client = WsClientBuilder::default()
+///          .custom_headers(&[Header { name: "Origin", value: "localhost".as_bytes() }])
```

Comment:
I think that's fine in this case, you mean like having a `Vec<(&str, &str)>` internally? 

but you like `custom_header` over `custom_headers` because it's more readable?